randomScripts/Closure/closurePlotter.py

```diff
@@ -490,23 +488,21 @@ def checkClosure(fullhisto, dicvar1, dicvar2, sigs):
 }
 
 
-inp = sys.argv[1] # "/eos/user/c/cericeci/www/SUEP/UL16APV/SR_2D/" 
+inp = sys.argv[1]
+
+first_bin = int(sys.argv[8])
+second_bin = int(sys.argv[9])
```

Comment:
Ok, but this invalidates the usage of sys.arg[3]... sys.argv[6]  back in L265-267, no?